x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/aggregation/X-ValueByTimestampAggregator.java.st

```diff
@@ -8,6 +8,12 @@
 package org.elasticsearch.compute.aggregation;
 
 // begin generated imports
+$if(BytesRef)$
+import org.apache.lucene.util.BytesRef;
+import org.elasticsearch.common.util.ObjectArray;
+import org.elasticsearch.compute.operator.BreakingBytesRefBuilder;
+import org.elasticsearch.common.breaker.CircuitBreaker;
+$endif$
```

Comment:
I'd import these unconditionally. You can keep doing this, but because the checkstyle rules need these to be sorted, you might end up in a maddening place.